docs/xls-65d-single-asset-vault/reference/transactions/vault-withdraw.md

```diff
@@ -0,0 +1,70 @@
+---
+seo:
+    description: Withdraws a specified number of assets from a vault in exchange for shares.
+labels:
+  - Transactions
+  - Single Asset Vault
+---
+
+# VaultWithdraw
+
+[[Source]](https://github.com/XRPLF/rippled/blob/9d619b9dc579c592f0560c1b40fd3c98d7587d23/src/xrpld/app/tx/detail/VaultWithdraw.cpp "Source")
+
+Withdraws assets from a vault or redeems vault shares for assets. The number of shares burned or assets received depends on the [exchange rate](../../concepts/single-asset-vault.md#exchange-algorithm), which adjusts based on the vault’s total assets and any [unrealized losses](../../concepts/single-asset-vault.md#paper-loss-unrealized-loss).
+
+{% admonition type="info" name="Note" %}
+The `VaultWithdraw` transaction does not respect the Permissioned Domain rules. In other words, any account that holds the shares of the vault can withdraw them. This is to avoid a situation where a depositor deposits assets to a private vault to then have their access revoked by invalidating their credentials, and thus loosing access to their funds.
```

Comment:
```suggestion
The `VaultWithdraw` transaction does not respect the Permissioned Domain rules. In other words, any account that holds the shares of the vault can withdraw them. This is to avoid a situation where a depositor deposits assets to a private vault to then have their access revoked by invalidating their credentials, and thus losing access to their funds.
```